### ISSUE TITLE
[Feature Request]: Disable Auto Translated Titles

### ISSUE THREAD

**Issue by paull27** [yuliskov/SmartTube#4225]
### Describe the feature you'd like to request

Hi,
first of all thank you so much 🙏🏻

For the feature request:
I don't like the youtube auto translated titles. I prefer to know the original video title. Right now, smarttube uses the main language in settings and translates video titles into that language.

### Describe the solution you'd like

A setting to disable translated video titles

### Describe alternatives you've considered

Switching the language to English translates German video titles into English. Also not a great solution

**Comment by tm-frs** [comment]
I totally agree with this, would love to see this happen (or just having a list of secondary languages that should not be translated, and all other languages being translated to your primary lqnguage. however, then an indication on whether the video is translated and from what language would be helpful). These settings maybe could be combined with audio translation, because i suppose you'd want the video title's language to match the audio's language, so i think having these options seperated would just add unnnecessary amount of options without any practical use compared to a unified Language/Translation option (but whatever, just my thoughts)

**Comment by fbnlsr** [comment]
I agree this would be a nice addition.

We've had with [Release 25.24 Stable](https://github.com/yuliskov/SmartTube/releases/tag/25.24s) the ability to disable AI audio translation. Title translation could be combined with this setting, to let people use original titles/audio.

**Comment by r3dicalnusz** [comment]
It still is so annoying and would be an amazing feature of SmartTube.

**Comment by olus85** [comment]
i really hate this translated titles, hate ai audio too. audio can be disabled, please add title to the settings.

i need to look at the thumbnails-text (if one existis) to check if its fit my language or not.

thank you for this great app devs! you are making the best job in this game 👍

**Comment by 0xA18** [comment]
Please do this, it’s so annoying!

**Comment by olus85** [comment]
yes, this is very annoying. every time i think its a video in my native language when a thumbnail is not showing foreign language text. after selecting non-ai audio, we should also be able to select non-ai translated titles. please add this feature.

**Comment by DivanoDova** [comment]
Please :)

**Comment by Jakke102** [comment]
Please, NewPipe is allready doing this, you need to catch up :)
just kidding, would be a nice addition!

**Comment by zaxcer** [comment]
YT started translating about 50% of YT titles since last month for me, the option to disable this annoying function would be great +1

**Comment by Lken-cmd** [comment]
+1

**Comment by marek-knappe** [comment]
+1

**Comment by mihe-hacs** [comment]
+1

**Comment by olus85** [comment]
why we need to wait 9 month and nothing is happening? is it that hard to implement? or is the app-owner not interested in this feature? any updates? its really very annoying when titles are translated and you dont know it.

**Comment by zaxcer** [comment]
> why we need to wait 9 month and nothing is happening? is it that hard to implement? or is the app-owner not interested in this feature? any updates? its really very annoying when titles are translated and you dont know it.

Dude, keep in mind this is a free open-source app created by people who dedicate their own time without getting paid. They deserve respect, because this isn’t some official user support team, it’s a community-driven project. I also get really frustrated with Google’s stupid features, like that annoying auto-translate that pops up everywhere. But that’s a reason to be pissed at Google, not at the SmartTube developers. We should actually be grateful for this app, since it gives us things the official app doesn’t even offer.

**Comment by olus85** [comment]
dude, im not to be pissed at our beloved devs here. just read my other comments in this topic, months ago. im just asking if its a programming problem or is it just not a high priority for them. english is not my native language, sorry for sounding rude. 

> > why we need to wait 9 month and nothing is happening? is it that hard to implement? or is the app-owner not interested in this feature? any updates? its really very annoying when titles are translated and you dont know it.
> 
> Dude, keep in mind this is a free open-source app created by people who dedicate their own time without getting paid. They deserve respect, because this isn’t some official user support team, it’s a community-driven project. I also get really frustrated with Google’s stupid features, like that annoying auto-translate that pops up everywhere. But that’s a reason to be pissed at Google, not at the SmartTube developers. We should actually be grateful for this app, since it gives us things the official app doesn’t even offer.

**Comment by IetIesAai** [comment]
Disabling translating the title seems already possible: settings—>user interface —>misc and toggle “unlocalized video titles”?

Would be great if that also worked forthr video descriptions (when you tap the i-icon at the bottom of a playing video) as that does sometimes seem to autotranslate.
